r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -21,6 +21,162 @@ skip_if_not_available("utf8proc")
 library(dplyr)
 library(stringr)
 
+test_that("paste, paste0, and str_c", {
+  df <- tibble(
+    v = c("α", "β", "γ"),
```

Comment:
I would advise against putting UTF-8 in this test file and just trust that the C++ library handles it correctly. Could just be outdated CRAN paranoia, but I've definitely had checks fail on some platform where it couldn't parse the test file because it had special characters in it (sounds like a Solaris thing right?). See also test-utf.R

Reply:
Changed to ASCII chars in c8a4ab2